lib/src/test/kotlin/io/bloco/faker/components/AvatarTest.kt

```diff
@@ -0,0 +1,47 @@
+package io.bloco.faker.components
+
+import io.bloco.faker.Faker
+import junit.framework.TestCase.assertFalse
+import org.junit.Before
+import org.junit.Test
+import kotlin.test.assertTrue
```

Comment:
sometimes you use junit.framework.TestCase.assertTrue other kotlin.test.assertTrue :) Yes I am picking on small things... better sleep with my 👁️ 👁️ opened tonight...